packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -798,7 +798,7 @@ class ToggleButtons extends StatelessWidget {
       return MergeSemantics(
         child: Semantics(
           container: true,
-          toggled: isSelected[index],
+          checked: isSelected[index],
```

Comment:
Semantics also has an `inMutuallyExclusiveGroup` attribute, which may or may not apply to ToggleButtons. Is it appropriate and feasible to include that?

Reply:
Ah looks like this would have to be implemented by the caller since it's not actually a primary feature of ToggleButtons, never mind.

Reply:
Indeed!